### DISCUSSION TITLE
The model give expected result LS dyna but doesn't give the same output in open Radioss solver

### DISCUSSION THREAD

**Discussion by okkarkyaw122** [OpenRadioss/OpenRadioss#3115] [Contributors]
Hello. 
I am a very fresh user of Open Radioss.  I can use Ls-Dyna really well. I am trying to simulate fracture by element erosion method. 
The model I am sharing works really well in LS dyna and gives out the expected result meaning I can check streess strain response, reactionary force etc. But when I run it in Radioss, the indentor in my model doesn't move and didn't have any response.
Excuse me if I made a mistake. I am a medical doctor by profession and did not have engineering background. I would appreciate any guidance. 
[3PBv2.zip](https://github.com/user-attachments/files/17620211/3PBv2.zip)

**Comment by okkarkyaw122** [comment]
![3PB screen shot](https://github.com/user-attachments/assets/3116939e-1645-43da-a50e-c70bc2f9dd44)
I want to check the color plot like this. Thank you.

**Reply by SyazwanSamad** [reply]
I have partially ran your model using OpenRadioss (for 0.8s) and it seems to be working fine. 
![image](https://github.com/user-attachments/assets/1edb0a43-22a9-467f-ae8f-38420a660b13)
here is another plot when I converted the OpenRadioss animations file to d3plot using Vortex-Radioss (but I cannot plot max princ strain due to it is unsupported by the converter)
![image](https://github.com/user-attachments/assets/229e0737-a3b6-48fc-9293-4ce58f959dd3)

**Reply by okkarkyaw122** [reply]
Thats really nice. However, I run the file in Radioss It runs perfectly. The problem is when I open the result. the indenter are not moving and there is no response at all. 
What could be the problems in this case. My computer CPU is AMD ryzen. Does it have anything to do with this CPU?
All my data in csv file are zero except time, time step and mass.

![image](https://github.com/user-attachments/assets/2bee214a-312c-4aa6-b862-10d9a9f05f05)

![image](https://github.com/user-attachments/assets/8f602dcf-22db-4ec1-943d-1803eee02e41)

**Reply by SyazwanSamad** [reply]
Can you provide us the OpenRadioss version (commit id) that you're using for this simulation? Are you using the latest stable version?

**Reply by okkarkyaw122** [reply]
This automatic release is built from commit id d17692d. I downloaded a few days ago.

**Reply by okkarkyaw122** [reply]
BTW it runs the uniaxial tensile test model from open radioss website. Here is the screenshot. 
![image](https://github.com/user-attachments/assets/609c75bd-9b87-416d-98c1-cce67a28bd68)

**Reply by okkarkyaw122** [reply]
I tried installing in a different computer. Apparently it analyzed the file without any error. But the Ann files only show the model with no movement and stress responses. (There is no displacement of the loading ring)

**Reply by okkarkyaw122** [reply]
> Can you provide us the OpenRadioss version (commit id) that you're using for this simulation? Are you using the latest stable version?

Hello. I solved the issue. I change the Boundary prescribe motion setting from VAD0= velocity to 3=displacement
It seems like VAD=0 doesnt work.
Thank you so much for your response.

**Reply by SyazwanSamad** [reply]
I downloaded the OpenRadioss with commit ID d17692d, and I'm having the same issues as yours. This feature is indeed broken, I think you can report this in the Issues section.

**Reply by okkarkyaw122** [reply]
> I downloaded the OpenRadioss with commit ID d17692d, and I'm having the same issues as yours. This feature is indeed broken, I think you can report this in the Issues section.

Thank you. I also completed the vortex radioos D3 plot conversion too.
